internal/cmdexport/cmdexport_test.go

```diff
@@ -276,6 +276,70 @@ pipeline {
         }
     }
 }
+`,
+	},
+	{
+		description: "exports a CircleCI workflow",
+		params: []string{
+			"resources",
+			"-w",
+			"circleci",
+		},
+		expected: `
+version: "2.1"
+orbs:
+    kpt:
+        executors:
+            kpt-container:
+                docker:
+                  - image: gongpu/kpt:latest
+        commands:
+            run-functions:
+                steps:
+                  - run: kpt fn run resources
+        jobs:
+            run-functions:
+                executor: kpt-container
+                steps:
+                  - setup_remote_docker
+                  - run-functions
+workflows:
+    main:
+        jobs:
+          - kpt/run-functions
+`,
+	},
+	{
+		description: "exports a CircleCI workflow with fn-path",
+		params: []string{
+			"resources",
+			"--fn-path",
+			"config/functions.yaml",
+			"-w",
+			"circleci",
+		},
+		expected: `
+version: "2.1"
+orbs:
+    kpt:
+        executors:
+            kpt-container:
+                docker:
+                  - image: gongpu/kpt:latest
+        commands:
+            run-functions:
+                steps:
+                  - run: kpt fn run resources --fn-path config/functions.yaml
+        jobs:
+            run-functions:
+                executor: kpt-container
+                steps:
+                  - setup_remote_docker
+                  - run-functions
```

Comment:
Is the step "run-functions" from the commands "run-functions"? Can you change the commands "run-functions" to "kpt-function-run"?